docs/pages/reference/mesheryctl/subcommands/mesheryctl-sys-update.md

```diff
@@ -0,0 +1,59 @@
+---
```

Comment:
@alphaX86 can we shift this to a template instead of repeating what is essentially the same page for each command?

Reply:
Hmm, this sounds reasonable though. I never thought of that!

Reply:
But things like `permalink` may need to be looked upon though. Not sure....